pkg/sink/codec/open/encoder.go

```diff
@@ -61,13 +64,23 @@ func (d *batchEncoder) Clean() {
 	}
 }
 
+func (d *batchEncoder) fetchColumnFlags(e *commonEvent.RowEvent) map[string]uint64 {
+	result, ok := d.columnFlagsCache[e.GetTableID()]
```

Comment:
Why cache column flag? It looks better to use a map to convert flag into columnFlag.

Reply:
the columnFlag is calculated by the flag, and cache it to prevent calculation on each message